onnx/defs/rnn/defs.cc

```diff
@@ -6,12 +6,20 @@
 using AttrType = onnx::OpSchema::AttrType;
 using namespace onnx;
 
+namespace onnx {
+
 std::function<void(OpSchema&)> RNNDocGenerator(const char* name) {
     return [=](OpSchema& schema) {
         schema.Attr("direction", "Specify if the RNN is forward, reverse, or bidirectional. "
                     "Must be one of forward (default), reverse, or bidirectional.",
                     AttrType::STRING);
         schema.Attr("hidden_size", "Number of neurons in the hidden layer", AttrType::INT);
+        schema.Attr("alpha", "Optional scaling values used by some activation functions.", AttrType::FLOATS);
+        schema.Attr("beta", "Optional scaling values used by some activation functions.", AttrType::FLOATS);
+        schema.Attr("output_sequence", "The sequence output Y is optional if 0. Default 0.", AttrType::INT);
```

Comment:
the description is not clear - and you probably need to change the Output description too

Reply:
Updated.